Makefile

```diff
@@ -31,7 +31,7 @@ dep: copy
 	build/odimra/makedep.sh
 
 build-containers: dep
-	cd build && docker-compose build
+	cd build && ./run_pre_reqs.sh && docker-compose build --force-rm --build-arg ETCD_USER_ID=1125 --build-arg ETCD_GROUP_ID=1125
```

Comment:
why is this hard coded?

Reply:
User ID and group ID of the host machine and in the container are enforced to match, so that the persistent volumes can have common ownership at both places and access to these persisted directories and files can be given only to that particular user.

Reply:
Removed hard-coding and the values are now read as environment variables.